src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { FlagsConfig, SfdxCommand } from '@salesforce/command';
-import { Logger, Messages, SfdxError } from '@salesforce/core';
```

Comment:
I guess `SfdxError` was not used?

Reply:
Yes, and switching over to eslint exposed this unused import.